### PR TITLE
#149/Create a Loading spinner component

### LINKED ISSUE #149: ⚛️ Create a Loading spinner component
Display this component instead of the `Loading...` text during API calls or actions that take time.

### PR DESCRIPTION
**Pull request by gbudau**
CSS spinner from https://loading.io/css

close #149

### CODE REVIEW

**Review by fizcamposancos — COMMENTED** [review]
I don't know if is the intra or what happens, bur I can not make login. I will try tomorrow again.
Anyway, in the components book looks nice.

**Comment by Damrod** [comment]
Can you put the spinner I'm the edit Avatar page as well, please?

**Comment by gbudau** [comment]
@damrod Done :rocket:

@fizcamposancos If the intra works, please check the OAuth secret. I had the same problem a few days ago. In my case, the secret expired, and I had to replace it with the new one.

File: webapp/src/pages/EditAvatarPage/EditAvatarPage.css [hunk 1]
```diff
@@ -39,6 +39,11 @@
     justify-items: center;
   }
 
+  .edit-avatar-loading {
+    grid-column: 1/-1;
```
Comment (Damrod) [inline webapp/src/pages/EditAvatarPage/EditAvatarPage.css:43]: I guess -1 is the index of... the last one?

**Review by Damrod — APPROVED** [review]
LGTM!

**Comment by gbudau** [comment]
Thank you!

**Pull request merged**